extensions/common/sql/common-sql/src/main/java/org/eclipse/dataspaceconnector/sql/translation/TranslationMapping.java

```diff
@@ -54,6 +54,25 @@ public String getStatement(String canonicalPropertyName) {
         return entry.toString();
     }
 
+    /**
+     * Returns the mapping for the input fieldName if it's a String
+     *
+     * @param fieldName The input
+     * @return The column name mapped to the input fieldName
+     */
+    public String getSimpleFieldMapping(String fieldName) {
```

Comment:
Why do we need this? Doesn't `getStatement` to the exact same thing?

Reply:
That was my first implementation since we have it. Then i got a security check complains on `JsonFieldMapping` which could open to SQL injection on order by.  So i figured it wanted a cast to simple type like `String`